core/net/src/hybrid/service.rs

```diff
@@ -73,7 +73,7 @@ async fn relay_addr(config: &Config) -> anyhow::Result<SocketAddr> {
     let (host, port) = &host_port
         .split_once(":")
         .context("Please use host:port format")?;
-    let ip = resolver::try_resolve_dns_record(&host).await;
+    let ip = resolver::try_resolve_dns_record(format!("udp://{}", host).as_str()).await;
```

Comment:
we should probably support `host`s with and without the scheme and apply the prefix conditionally